### ISSUE TITLE
Key press does not work on russian keyboard on OS X 10.12.6

### ISSUE THREAD

**Issue by undsett** [BlueM/cliclick#66]
I'm using cliclick to test hotkeys in my app. For example it presses control+shift+v. And the test runs on english and russian keyboards. But now then i run it on mac with 10.12.6 on board I get only control+shift when i switch to russian. Button "V" is ignored. It worked in earlier versions of OS X.

**Comment by BlueM** [comment]
What is the exact command you are executing?

And: are you sure there is no output from cliclick? I am asking as it sounds like cliclick is launched from a script, and depending on where/how the scripts is in your toolchain, output from cliclick may have been unnoticed.

**Comment by undsett** [comment]
The command is : 
cliclick -w 150 kd:cmd kd:shift t:v ku:shift ku:cmd
and the script does not end until cliclick is working. As a fact it's the same code that executes tests on english and russian keyboards,  it just changes language and runs again.

**Comment by BlueM** [comment]
On macOS 12.6, there are 3 Russian keyboards: “Russian”, “Russian – PC” and “Russian – Phonetic”. Which one are you having the problem with?

**Comment by undsett** [comment]
I use “Russian” but tested all three too and have the same problems with all of them.

**Comment by BlueM** [comment]
Testing again, it did not work on all 3 keyboards. I.e.: same result that you got, although pretty strange that this is a different result than I got 6 days ago …

Anyway, I compared the behavior running 10.12.6 and 10.11.6 (on another Mac), and both did not work using the keyboard named “Russian”. However, during testing, I learned that on all these 3 Russian keyboards, the layout changes from cyrillic to latin characters when pressing `⌘` – and by that, the usual shortcuts such as `⌘v` work. (Which can be observed when watching `/System/Library/Input Methods/KeyboardViewer.app` while experimenting.) As cliclick basically does not care about characters, but about the physical keys, instead of `t:v`, I simply tried the cyrillic character I got when I hit what is the `v` key on my Western keyboard – and with that `t:м`, it worked. (I only tried it with `⌘v`, but I guess the principle should be the same with other combinations.)

However, I have no idea why `t:v` might have worked for you earlier.
